src/utils/api.js

```diff
@@ -10,11 +10,23 @@ export const getArticleById = (article_id) => {
   })
 }
 export const getArticles = (article_id) => {
-    let path = '/articles';
-    if (article_id){
-      path += `?article_id=${article_id}`;
-    }
-    return articlesApi.get(path).then(({ data }) => {
+    // let path = '/articles';
+    // if (article_id){
+    //   path += `{?article_id=${article_id}`;
+    // }
+    return articlesApi.get('/articles').then(({ data }) => {
       return data
     });
-};
\ No newline at end of file
+}
+ export const GetTopicsApi = () => {
+   return articlesApi.get("/topics").then(({ data }) => {
+     return data.topics;
+   })
+ }
+ export const getTopicsApi = (topic) => {
```

Comment:
I would rename this function to be something like getArticlesByTopic so that someone can understand what the function does and makes it more distinguished from the function above it :)